### PR TITLE
Anova Analysis

### LINKED ISSUE #185: ANOVA

### PR DESCRIPTION
**Pull request by sgratzl**
based on #191 

closes #185 

implements ANOVA analysis based on the given R code

Screenshot from 2019-09-16 08-48-07

notes
 * please rebuild your docker image

### CODE REVIEW

File: devops/metabulo/R/analyses.R [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#' Basic PCA plot
+#'
+#' Uses MetaboAnalystR's PlotPCAPairSummary method
+
+anova_turkey_adjustment <- function(measurements, groups) {
```
Comment (jbeezley) [inline devops/metabulo/R/analyses.R]: I believe this should be "tukey".

File: metabulo/views.py [hunk 2]
```diff
@@ -489,3 +489,21 @@ def get_wilcoxon_test(validated_table, zero_method, alternative):
...
+    'group_column': fields.Str()
+})
+@load_validated_csv_file
+def get_anova_test(validated_table, group_column=None):
+    measurements = validated_table.measurements
+    groups = validated_table.groups
+
+    group = groups.iloc[:, 0] if group_column is None else groups.loc[:, group_column]
+    if group is None:
+        abort(404, 'invalid group column')
```
Comment (jbeezley) [inline metabulo/views.py]: This should throw a `marshmallow.ValidationError`, which will return a properly formatted 400 error.

File: devops/metabulo/R/analyses.R (hunk 1 above)
Comment (sgratzl) [reply devops/metabulo/R/analyses.R]: that happens when you think of turkey to eat :-D

**Comment by subdavis** [comment]
Closing in favor of #212

**Pull request closed without merging**